proxy/coco/tests/common.rs

```diff
@@ -146,6 +138,7 @@ pub fn radicle_project(path: PathBuf) -> project::Create<PathBuf> {
     }
 }
 
+#[allow(dead_code)]
```

Comment:
delete?

Reply:
It's used in the integration tests. Haven't found a way for the compiler not to claim it's dead code.